scalableminds/webknossos
# Add optional param skipVolumeData to annotation download requests

### URL of deployed dev instance (used for testing):
- https://skipvolumedata.webknossos.xyz

### Steps to test:
- create volume annotation
- `/api/annotations/Explorational/ANNOTATION_ID/download?skipVolumeData=true` should give you zip with nml only, no `data.zip`
- should default to false
- should also work for compound projects/tasks etc

### Issues:
- supports workaround for #3874 

------
